src/routes/profile.tsx

```diff
@@ -14,10 +14,17 @@ import { S3Agent } from '../clients/s3/S3Agent';
 import { PatreonAgent } from '../clients/patreon/patreonAgent';
```

Comment:
Let's make a new folder called `profile`, keep the profile.ts here, and then have it import `profile/profile.ts` and `profile/avatar.ts`